### ISSUE TITLE
[TF 2.0 API Docs] `tf.keras.callbacks.LearningRateScheduler` (Very small update)

### ISSUE THREAD

**Issue by shun-lin** [tensorflow/tensorflow#34975]
## URL(s) with the issue:

Please provide a link to the documentation entry, for example:
Doc Link:
https://www.tensorflow.org/api_docs/python/tf/keras/callbacks/LearningRateScheduler
Code Link:
https://github.com/tensorflow/tensorflow/blob/r2.0/tensorflow/python/keras/callbacks.py#L1311-L1358

### Parameters

The next API for `scheduler` parameter of `LearningRateScheduler` takes in 2 parameters, `epoch` and `lr` (learning rate) instead of just `epoch`, this is evident in the `on_epoch_begin` of the `LearningRateScheduler` method.

The documentation for this method is still outdated, the docs and the example code still shows the `scheduler` function takes in only `epoch` instead of both `epoch` and `lr`. I think the doc should be updated to reflect the new API.

Proposed change to the doc:

1) update the description of `scheduler`: 

`
schedule: a function that takes an epoch index as input (integer, indexed from 0) and current learning rate and returns a new learning rate as output (float).
`
(copied from the doc from keras.io)

2) update the example usage to include a `scheduler` that utilize the current learning rate as well.

I hope this is helpful! Happy to contribute if needed.

### Submit a pull request? Yes, if this should be updated.

Are you planning to also submit a pull request to fix the issue? See the docs
contributor guide: https://www.tensorflow.org/community/contribute/docs,
docs API guide: https://www.tensorflow.org/community/contribute/docs_ref and the
docs style guide: https://www.tensorflow.org/community/contribute/docs_style

**Comment by shun-lin** [comment]
fixed :)
